src/core/stdcpp/new_.d

```diff
@@ -14,8 +14,6 @@ module core.stdcpp.new_;
 import core.stdcpp.xutility : __cpp_sized_deallocation, __cpp_aligned_new;
 import core.stdcpp.exception : exception;
 
-@nogc:
```

Comment:
Every function between here and where `@nogc:` is re-added is either a template or explicitly annotated `@nogc`.